File: KahveQRKodOlusturucuJava-calismalarim/app/src/test/java/com/example/qrkodolusturucujava/usecasesTest/IndirimYuzdeHesaplaTest.java
package com.example.qrkodolusturucujava.usecasesTest;

import com.example.qrkodolusturucujava.usecases.IndirimHesapla;

import org.junit.Assert;
import org.junit.Test;

public class IndirimYuzdeHesaplaTest {

    @Test
    public void indirimYuzdeHesaplaTest() {
        IndirimHesapla indirimHesapla = new IndirimHesapla();
        String indirimYuzdesi = indirimHesapla.indirimYuzdeHesapla(100.0, 90.0);
        Assert.assertEquals("10,0", indirimYuzdesi);
    }
}

File: KahveQrOlusturucuKotlin-calismalarim/app/src/test/java/com/example/qrkodolusturucu/usecasesTEST/IndirimYuzdeHesaplaTest.kt
package com.example.qrkodolusturucu.usecasesTEST

import com.example.qrkodolusturucu.usecases.indirimYuzdeHesapla
import org.junit.Assert
import org.junit.Before
import org.junit.Test

class IndirimYuzdeHesaplaTest {

    @Test
    fun indirimYuzdeHesaplaTest(){
        var indirimYuzdesi:String = indirimYuzdeHesapla(100.0,90.0)
        Assert.assertEquals(indirimYuzdesi,"10,0")
    }

}